ubuntu/standard/5.0/Dockerfile

```
@@ -298,11 +298,15 @@ RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/bin -
 #****************      END PHP     ****************************************************
 
 #****************     GOLANG     ****************************************************
-ENV GOLANG_15_VERSION="1.15.12"
-ENV GOLANG_16_VERSION="1.16.4"
+ENV GOLANG_15_VERSION="1.15.15"
+ENV GOLANG_16_VERSION="1.16.15"
+ENV GOLANG_17_VERSION="1.17.8"
+ENV GOLANG_18_VERSION="1.18"
```

There is a PR for `goenv` supporting `1.18.1` opened already:

```suggestion
ENV GOLANG_18_VERSION="1.18.1"
```

Since this PR has been opened for a longer while now, maybe it's not much of a timing gamble to suggest this?

---

@friimaind Glad that you like the idea - could you update the PR description as well (just for clarity)?

`Add support for golang: 1.18.1`